heterogeneity/Python_Order

```diff
@@ -1,103 +1,60 @@
 This file will document how to install and run the Python Scripts in the RamosRivera folder necessary for this project.
+This enviroment is only been created for the G drive in my personal computer. 
```

Comment:
@JorgeValde This is confusing. What do you mean by personal computer? Isn't it true that if something is on the G:drive, everyone can access it? G: is the shared drive.